src/rules/seczetta-grab-risk-score.js

```diff
@@ -0,0 +1,223 @@
+          'Accept': 'application/json'
+        },
+      });
+
+      //if the user isnt found via the advanced search. A 
+    	if( profileResponse.data.profiles.length === 0 ) {
+        console.log("Profile not found. Empty Array sent back!");
+  	 		if( configuration.SECZETTA_AUTHENTICATE_ON_ERROR && configuration.SECZETTA_AUTHENTICATE_ON_ERROR === "true" ) {
+          return callback(null, user, context);
+        }
+        return callback(new UnauthorizedError("Error retrieving Risk Score."));
+  		}
+    
+  } catch (profileError) {
+    // Swallow risk scope API call, default is set to highest risk below.
+    console.log(`Error while calling Profile API: ${profileError.message}`);
+    if( configuration.SECZETTA_AUTHENTICATE_ON_ERROR && configuration.SECZETTA_AUTHENTICATE_ON_ERROR === "true" ) {
+      return callback(null, user, context);
+    }
+    return callback(new UnauthorizedError("Error retrieving Risk Score."));
+  }
+
+  //Should now have the profile in profileResponse. Lets grab it.
+  let objectId = profileResponse.data.profiles[0].id;
+  console.log(objectId);
+
+  let riskScoreRequestUrl = configuration.SECZETTA_BASE_URL + '/risk_scores?object_id=' + objectId;
+
+  try {
+      riskScoreResponse = await axios.get(riskScoreRequestUrl,{
+      headers: {
+        'Content-Type':'application/json',
+        'Authorization': 'Token token='+configuration.SECZETTA_API_KEY,
+        'Accept': 'application/json'
+      },
+    });
+  } catch (riskError) {
+    // Swallow risk scope API call, default is set to highest risk below.
+    console.log(`Error while calling Risk Score API: ${riskError.message}`);
+    if( configuration.SECZETTA_AUTHENTICATE_ON_ERROR && configuration.SECZETTA_AUTHENTICATE_ON_ERROR === "true" ) {
+      return callback(null, user, context);
+    }
+    return callback(new UnauthorizedError("Error retrieving Risk Score."));
+  }
+
+  //Should now finally have the risk score. Lets add it to the user
+  var riskScoreObj = riskScoreResponse.data.risk_scores[0];
+  var overallScore = riskScoreObj.overall_score;
+
+  // Default risk value is set to highest if API fails or no score returned.
+  //var riskScore = typeof apiResponse.riskScore === "number" ? riskScore : 100;
+
+  const allowableRisk = parseInt(configuration.SECZETTA_ALLOWABLE_RISK, 10);
+  const maximumRisk = parseInt(configuration.SECZETTA_MAXIMUM_ALLOWED_RISK, 10);
+  
+  //if risk score is below the maxium risk score but above allowable risk: Require MFA
+  if (allowableRisk && overallScore > allowableRisk && overallScore < maximumRisk) {
+    console.log(
+      `Risk score ${overallScore} is greater than maximum of ${allowableRisk}. Prompting for MFA`
+    );
+    context.multifactor = {
+      provider: 'any',
+      allowRememberBrowser: false
+    };
+    return callback(null, user, context);
+  
+  }
+
+  //if risk score is above the maxium risk score: Fail authN
+  if (maximumRisk && overallScore > maximumRisk) {
```

Comment:
```suggestion
  if (maximumRisk && overallScore => maximumRisk) {
```

Seems like this is a bit more accurate but I could be wrong.